### PR TITLE
Clarify k_i construcion, remove ednote

### PR DESCRIPTION
**Pull request by wussler**
Remove ednote creating confusion about pre-hashing of the `k_i`

### CODE REVIEW

File: draft-ounsworth-cfrg-kem-combiners.mkd [hunk 1]
```diff
@@ -294,34 +294,22 @@ The shared secret `ss` MAY be used directly as a symmetric key, for example as a
...
 ~~~
 k_i = ss_i || ct_i
 ~~~
 
-For all other cases, it is REQUIRED to concatenate and hash them first:
+If `ss_i` or `ct_i` are not guaranteed to have constant length, it is REQUIRED to concatenate and hash the resulting string prior to inclusion in the overall construction described in {{tab-kemCombiner}}:
 
 ~~~
 k_i = H(ss_i || ct_i)
 ~~~
```
Comment (wussler) [inline draft-ounsworth-cfrg-kem-combiners.mkd]: Use length encoding instead of hashing, copy it from KMAC, reference it

Right-encoded

Comment (wussler) [reply draft-ounsworth-cfrg-kem-combiners.mkd]: Swap `ct_i || ss_i` to allow embedded systems can first slam in ciphertext

Comment (wussler) [reply draft-ounsworth-cfrg-kem-combiners.mkd]: Allow option of length encoded or fixed length

File: draft-ounsworth-cfrg-kem-combiners.mkd [hunk 2]
```diff
@@ -292,43 +292,70 @@ where:
...
+
+The `fixedInfo` string MUST have a definite structure depending on the protocol where all parts strictly defined by the protocol specification.
+
+~~~
+fixedInfo = fixedInfo || s
+~~~
+
+Each fixed-length input string `f` MAY be directly used as input:
 
-The `fixedInfo` string MUST have a definite structure depending on the protocol where all parts are fixed length. This prevents a variable length structure from creating collisions between two different instances.
```
Comment (falko-strenzke) [inline draft-ounsworth-cfrg-kem-combiners.mkd:330]: I think requiring fixed length as MUST can be too restritive. Imagine the case where specifiers used for the characterization of an instance have no length bounds per se.

Comment (wussler) [reply draft-ounsworth-cfrg-kem-combiners.mkd:330]: This sentence has been in fact removed, I also agree we can allow variable-length, as long as they are suffixed with `rlen`

**Comment by falko-strenzke** [comment]
Oh yes, still too early for me this morning I guess ;-)

**Comment by ounsworth** [comment]
The other important change in this PR is this one to change the order, put `ct_i` first to be more streaming-friendly.

```
+ k_i = ct_i || ss_i
- k_i = H(ss_i || ct_i)
```

**Comment by ounsworth** [comment]
This is a great change. Thanks Aron! I will merge as soon as the comments above are resolved. Should I then push the new version to datatracker?

**Comment by wussler** [comment]
> This is a great change. Thanks Aron! I will merge as soon as the comments above are resolved. Should I then push the new version to datatracker?

Hey @ounsworth, the comment above was already resolved - just forgot to mark it as such.

If it looks ready to be merged, please feel free to do so.
Before releasing a new version on the datatracker I'd like to change the term `multi-PRF` to `Split-key PRF` as mentioned from Giacon et al.

**Pull request merged**